### DISCUSSION TITLE
Vinnicombe gap metric

### DISCUSSION THREAD

**Discussion by jpreiss** [python-control/python-control#648] [Ideas]
The metric known as Vinnicombe gap, v-gap, or ν-gap is a useful tool for measuring the "distance" between transfer functions. As far as I can tell, there is no available Python implementation. The only implementation I can find anywhere is in Matlab's robust toolbox.

Would this be of interest to the community?

Vinnicombe, G (1993). Frequency domain uncertainty and the graph topology.
IEEE Transactions on Automatic Control, 38. pp. 1371-1383.
doi: 10.1109/9.237648

**Comment by murrayrm** [comment]
Definitely of interest.
